### ISSUE TITLE
Extra Social Media

### ISSUE THREAD

**Issue by jakelever** [hackseq/hackseq_2017#18]
Hey @abaghela  , I was just perusing the reddit [/r/bioinformatics](https://www.reddit.com/r/bioinformatics/) subreddit and thought there were a few other places we could advertise for team-leader recruitment and general hackseq awareness. Do we have a reddit account for this?

And how about the [Biostars forum](https://www.biostars.org/t/Forum/) or perhaps somewhere else on Biostars? Maybe [seqanswers.com](http://seqanswers.com/) too?

Does anyone know any other relevant bioinformatics website we should post too?

**Comment by abaghela** [comment]
Yes, I think we do have a reddit account. I will try to advertise at these places now!
